metron-deployment/amazon-ec2/playbook.yml

```diff
@@ -62,6 +62,7 @@
     - include: tasks/check-volume.yml vol_name=xvda vol_src=/dev/xvda vol_size={{ xvda_vol_size }}
   tags:
     - ec2
+    - ec2-mount
 
```

Comment:
I think the timely part is the wait, so --skip-tags="wait" usually works for me, but if you really want this, couldn't it be part of a JIRA that addresses performance rather than the snort deploy bug?